### PR TITLE
MC-32651: A Gift Card with minimum value of an open amount, greater than maximum value, can be created

### PR DESCRIPTION
**Pull request by SmVladyslav**
### Related Pull Requests
https://github.com/magento/partners-magento2ee/pull/494

### Fixed Issues (if relevant)

1. Fixes magento/partners-magento2ee#493

### Manual testing scenarios (*)

1. ...
2. ...

### Contribution checklist (*)
 - [ ] Pull request has a meaningful description of its purpose
 - [ ] All commits are accompanied by meaningful commit messages
 - [ ] All new or changed code is covered with unit/integration tests (if applicable)
 - [ ] All automated tests passed successfully (all builds are green)

### CODE REVIEW

**Comment by SmVladyslav** [comment]
@magento run all tests

**Review by BarnyShergold — APPROVED** [review]

**Comment by magento-engcom-team** [comment]
Hi @BarnyShergold, thank you for the review.
[ENGCOM-8771](https://jira.corp.magento.com/browse/ENGCOM-8771) has been created to process this Pull Request
:eight_spoked_asterisk: @BarnyShergold, could you please add one of the following labels to the Pull Request?

| Label | Description |
|--------------------------------|-----------------------------------------------------------------|
| ```Auto-Tests: Covered``` | All changes in Pull Request is covered by auto-tests |
| ```Auto-Tests: Not Covered``` | Changes in Pull Request requires coverage by auto-tests |
| ```Auto-Tests: Not Required``` | Changes in Pull Request does not require coverage by auto-tests |

**Comment by SmVladyslav** [comment]
@magento run Sample Data Tests EE

File: app/code/Magento/Ui/view/base/web/js/lib/validation/rules.js [hunk 1]
```diff
@@ -854,6 +854,26 @@ define([
             },
             $.mage.__('Please enter a value greater than or equal to {0}.')
         ],
+        'validate-range-from': [
```
Comment (zakdma) [inline app/code/Magento/Ui/view/base/web/js/lib/validation/rules.js]: Could you do the same as validate-date-range?
https://github.com/magento-commerce/magento2ce/blob/2.4-develop/lib/web/mage/validation.js#L1055

**Review by zakdma — CHANGES_REQUESTED** [review]

**Comment by SmVladyslav** [comment]
@magento run all tests

File: app/code/Magento/Ui/view/base/web/js/lib/validation/rules.js (hunk 1 above)
Comment (SmVladyslav) [reply app/code/Magento/Ui/view/base/web/js/lib/validation/rules.js]: Fixed

**Comment by engcom-Oscar** [comment]
QA fail.

Validation fails if input in filed “Open Amount From” value with comma but smaller then in field 'To', and click 'Save'. 

Screenshot from 2021-02-23 16-18-26

**Pull request closed without merging**